{
  "use_super4pcs": true,
  "use_icp": true,
  "process" :
  {
    "downsample_leaf_size": 0.01,
    "normal_SearchRadius": 0.01,
    "features_SearchRadius": 0.025
  }
}
